Local test_standardise failures

Three unit tests in `test_standardise.py` fail locally (but pass on GitHub Actions):
- `test_standardise_reorientation`
- `test_standardise_downsampling `
  - `[isotropic input - scale by 0.5, 0.5, 0.5]`
  - `[anisotropic input - scale by 0.5, 0.2, 0.1]`

This is probably caused by `image.header.get_zooms()` returning floats with slight precision differences vs. the expected outputs.

Using `np.allclose` where `image.header.get_zooms()` is compared against expected values, should resolve the issue. 
```
assert np.allclose(image.header.get_zooms(), expected)
```

**Error message:**
> FAILED tests/test_unit/test_standardise.py::test_standardise_reorientation - assert (0.01, 0.01, 0.01) == (0.01, 0.01, 0.01)
> FAILED tests/test_unit/test_standardise.py::test_standardise_downsampling[isotropic input - scale by 0.5, 0.5, 0.5] - assert (0.02, 0.02, 0.02) == (0.02, 0.02, 0.02)
> FAILED tests/test_unit/test_standardise.py::test_standardise_downsampling[anisotropic input - scale by 0.5, 0.2, 0.1] - assert (0.02, 0.02, 0.02) == (0.02, 0.02, 0.02)

**Note**:
Weirdly, this fails on my Windows machine locally, but not on @K-Meech's.

--- a/tests/test_unit/test_standardise.py
+++ b/tests/test_unit/test_standardise.py
@@ -425,7 +425,7 @@ def test_standardise_downsampling(request, source_csv, expected_output_size):
     image_path = subject_dir / "sub-b_res-20x20x20um_origin-asr.nii.gz"
     mask_path = subject_dir / "sub-b_res-20x20x20um_mask_origin-asr.nii.gz"
 
-    expected_output_vox_sizes_mm = (
+    output_vox_sizes_mm = (
         output_vox_size * 0.001,
         output_vox_size * 0.001,
         output_vox_size * 0.001,
@@ -434,6 +434,6 @@ def test_standardise_downsampling(request, source_csv, expected_output_size):
     for output_path in [image_path, mask_path]:
         image = load_nii(output_path, as_array=False)
         np.testing.assert_allclose(
-            image.header.get_zooms(), expected_output_vox_sizes_mm
+            image.header.get_zooms(), output_vox_sizes_mm
         )
         assert image.shape == expected_output_size

